swagger2locustio/generators/base_generator.py

```diff
@@ -78,6 +78,7 @@ def generate_locustfiles(self, swagger_data: dict) -> None:
             class_file_path = self.results_path / self.tests_path / test_class.file_path
             class_file_path.mkdir(parents=True, exist_ok=True)
             file_name = f"{test_class.file_name}.py"
+            class_constants = list(sorted(class_constants))
```

Comment:
Unfortunately, mypy does not agree with such reassignment:
```
mypy.....................................................................Failed
- hook id: mypy
- exit code: 1

swagger2locustio/generators/base_generator.py:81: error: Incompatible types in assignment (expression has type "List[Constant]", variable has type "Set[Constant]")
```

Unfortunately, creating a new variable does not match pylint requirements:

```
pylint...................................................................Failed

swagger2locustio/generators/base_generator.py:61:4: R0914: Too many local variables (16/15) (too-many-locals)
```